### PR TITLE
Create unrestrainfly-chengdu.md

### PR DESCRIPTION
**Pull request by unrestrainfly**
added the room type and the owner

### CODE REVIEW

**Comment by unrestrainfly** [comment]
bug fixed

File: unrestrainfly-chengdu.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Meeting room type
+ - Collabrate room
```
Comment (unrestrainfly) [inline unrestrainfly-chengdu.md:2]: can you explain a little more about this room.

**Comment by rwim2** [comment]
This is a comment - just to test it out

**Comment by unrestrainfly** [comment]
test  merge

**Pull request merged**